gnupg/_util.py

```diff
@@ -41,6 +41,17 @@
 except ImportError:
     from cStringIO import StringIO
 
+# Import the StringIO class from the StringIO module since it is a
+# commonly used stream class. It is distinct from either of the
+# StringIO's that may be loaded in the above try/except clause, so the
+# name is prefixed with an underscore to distinguish it.
+from StringIO import StringIO as _StringIO
```

Comment:
@ttanner Thanks for testing that! I figured the test might need some additional conditionals to work properly on Python 3, but wanted to wait to get the go-ahead that this overall approach was indeed the right one before investing any more time in this PR.